oak-core/src/test/java/org/apache/jackrabbit/oak/query/QueryInformationLoggerTest.java

```diff
@@ -0,0 +1,143 @@
+package org.apache.jackrabbit.oak.query;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.jackrabbit.oak.commons.junit.LogCustomizer;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+
+public class QueryInformationLoggerTest {
+    
+    private static final Logger LOG = LoggerFactory.getLogger(QueryInformationLoggerTest.class);
```

Comment:
i might be mistaken.... but i haven't seen this LOG field used.... if it's not -> drop.